coverage/htmlfiles/coverage_html.js

```diff
@@ -81,7 +81,31 @@ function sortColumn(th) {
         .forEach(tr => tr.parentElement.appendChild(tr));
 
     // Save the sort order for next time.
-    localStorage.setItem(coverage.INDEX_SORT_STORAGE, JSON.stringify({column, direction}));
+    if (th.id !== "function_or_class") {
```

Comment:
I don't want to use `function_or_class` as the id.  I've been careful throughout the function/class implementation to generalize to "region", and in the reports I simply call it "column2" (though now that I type this, "region" would have been better).